packages/fx-core/src/core/index.ts

```diff
@@ -489,6 +492,7 @@ export class FxCore implements Core {
   @hooks([
     ErrorHandlerMW,
     ConcurrentLockerMW,
+    ProjectMigratorMW,
```

Comment:
@meifans have we confirmed whether this middleware will impact other migration logic if the migration logic is executed?

Reply:
@blackchoey  Has confirmed it with multi-env.